src/overlays/actors/ovl_En_Bjt/z_en_bjt.c

```diff
@@ -15,10 +15,32 @@ void EnBjt_Destroy(Actor* thisx, PlayState* play);
 void EnBjt_Update(Actor* thisx, PlayState* play);
 void EnBjt_Draw(Actor* thisx, PlayState* play);
 
-void func_80BFDA48(EnBjt* this, PlayState* play);
-void func_80BFDAE8(EnBjt* this, PlayState* play);
+void EnBjt_Talk(EnBjt* this, PlayState* play);
+void EnBjt_Wait(EnBjt* this, PlayState* play);
+
+#define TOILET_HAND_STATE_TALKING (1 << 3)   // Actually talking to Player
+#define TOILET_HAND_STATE_TEXTBOX (1 << 4)   // whenever a textbox is on screen
+#define TOILET_HAND_STATE_APPEARING (1 << 7) // Appearing
+#define TOILET_HAND_STATE_VANISHING (1 << 8) // Vanishing
+#define TOILET_HAND_STATE_VISIBLE (1 << 9)   // Out
+
+static u8 sScheduleScript[] = {
+    /* 0x00 */ SCHEDULE_CMD_CHECK_NOT_IN_SCENE_S(SCENE_YADOYA, 0x11 - 0x04),
+    /* 0x04 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(0, 0, 6, 0, 0x0B - 0x0A),
+    /* 0x0A */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x0B */ SCHEDULE_CMD_RET_TIME(0, 0, 6, 0, 1),
+    /* 0x11 */ SCHEDULE_CMD_RET_NONE(),
+};
```

Comment:
Thoughts on using an enum for the results like in #825.

Reply:
It'd just be PRESENT or ABSENT, not sure it's needed here.

Reply:
If we do it elsewhere I'd prefer to do it everywhere, even for small cases like this. One thing I like about the enum is you can actually see what the actor does based solely on the schedule script. Though I'll admit, we haven't seen it yet with many different results (Sakon for example has a ton).